examples/todos/src/actions/index.js

```diff
@@ -1,22 +1,26 @@
+// @flow
+
 let nextTodoId = 0
-export const addTodo = text => ({
+export const addTodo = (text: string) => ({
   type: 'ADD_TODO',
   id: nextTodoId++,
   text
 })
 
-export const setVisibilityFilter = filter => ({
+export const VisibilityFilters = {
+  SHOW_ALL: 'SHOW_ALL',
+  SHOW_COMPLETED: 'SHOW_COMPLETED',
+  SHOW_ACTIVE: 'SHOW_ACTIVE'
+}
+
+export const setVisibilityFilter = (filter: typeof VisibilityFilters) => ({
```

Comment:
Why do we need a `typeof` here? 😄

Reply:
Because VisibilityFilters is being a object literal. To get the type of value I use ```typeof```.